.gitignore

```diff
@@ -0,0 +1 @@
+infrastructure/patent_watch.tf
```

Comment:
Why you need to ignore this file?

Reply:
I just created this file to check my code using s3 bucket before using in dynamodb after I get my api